src/Microsoft.PowerShell.Commands.Management/commands/management/Service.cs

```diff
@@ -1681,189 +1611,137 @@ protected override void ProcessRecord()
-
-                                    if ((dependentServices != null) && (dependentServices.Length > 0))
-                                    {
-                                        WriteNonTerminatingError(service, null, "ServiceHasDependentServicesNoForce", ServiceResources.ServiceHasDependentServicesNoForce, ErrorCategory.InvalidOperation);
-                                        continue;
-                                    }
-
-                                    ServiceController[] servicedependedon = service.ServicesDependedOn;
-
-                                    if ((servicedependedon != null) && (servicedependedon.Length > 0))
-                                    {
-                                        WriteNonTerminatingError(service, null, "ServiceIsDependentOnNoForce", ServiceResources.ServiceIsDependentOnNoForce, ErrorCategory.InvalidOperation);
-                                        continue;
-                                    }
-                                    //stop service,give the force parameter always true as we have already checked for the dependent services
-                                    //Specify NoWait parameter as always false since we are not adding this switch to this cmdlet
-                                    DoStopService(service, true, true);
+                                    WriteNonTerminatingError(service, null, "ServiceIsDependentOnNoForce", ServiceResources.ServiceIsDependentOnNoForce, ErrorCategory.InvalidOperation);
+                                    return;
                                 }
-                            }
-                            else if (Status.Equals("Paused", StringComparison.CurrentCultureIgnoreCase))
-                            {
-                                if (!service.Status.Equals(ServiceControllerStatus.Paused))
-                                    //pause service
-                                    DoPauseService(service);
+                                //stop service,give the force parameter always true as we have already checked for the dependent services
+                                //Specify NoWait parameter as always false since we are not adding this switch to this cmdlet
+                                DoStopService(service, true, true);
                             }
                         }
-                        if (PassThru.IsPresent)
+                        else if (Status.Equals("Paused", StringComparison.CurrentCultureIgnoreCase))
                         {
-                            //to display the service,refreshing the service would not show the display name after updating
-                            ServiceController displayservice = new ServiceController(Name, ServiceComputerName);
-                            WriteObject(displayservice);
+                            if (!service.Status.Equals(ServiceControllerStatus.Paused))
+                                //pause service
```

Comment:
This comment is unnecessary.

Reply:
I didn't change this one. I will remove it.